lib/utils.pm

```diff
@@ -101,6 +102,10 @@ sub unlock_if_encrypted {
     }
 }
 
+sub turn_off_packagekit() {
+    x11_start_program("systemctl stop packagekit.service");
```

Comment:
that is expected behavior on live system, there is no root password required on live system, thus the policykit pop-up and approved/closed very soon as no password needed.